cou/apps/auxiliary.py

```diff
@@ -196,35 +283,8 @@ def pre_upgrade_steps(
         :rtype: list[PreUpgradeStep]
         """
         steps = super().pre_upgrade_steps(target, units)
-        # Since auto restart is disabled, we don't know the if the service
-        # has pending events or not, so we want to run `run-deferred-hooks`
-        # action to clear the events before performing upgrade. If there
-        # are no pending events, this step should be a no-op, so it's safe
-        # to run anyway
```

Comment:
Could you re-instate this information, perhaps in the docstring of the new `get_run_deferred_hooks_and_restart...` methods? :)

Reply:
🙆🏼‍♂️